README.md

```diff
@@ -2,13 +2,19 @@
 Ingestion of GARD into OWL and SemanticSQL.
 
 ## Prerequisites
-* Python 3.9+
-* [robot](http://robot.obolibrary.org/)
+Python is a dev dependencies. It's not needed to run the docker containers, but needed for local development situations 
```

Comment:
## Docs
- About dev dependencies
- Added docker stuff to prereqs
- Added `sh run.sh` to commands